cluster-provision/gocli/cmd/run.go

```diff
@@ -373,8 +373,20 @@ func run(cmd *cobra.Command, args []string) (err error) {
 	wg := sync.WaitGroup{}
 	wg.Add(int(nodes))
 	// start one vm after each other
+
```

Comment:
please drop the line, the comment should not be far away from the block it comments

Reply:
Done